client/layout/guided-tours/positioning.js

```diff
@@ -156,8 +156,9 @@ function validatePlacement( placement, target ) {
 		: placement;
 }
 
-function scrollIntoView( target ) {
-	const container = getScrollableSidebar();
+function scrollIntoView( target, scrollContainer ) {
+	// TODO(lsinger): consider replacing with http://yiminghe.me/dom-scroll-into-view/
+	const container = scrollContainer || getScrollableSidebar();
```

Comment:
Does this work? In the config, I see `scrollContainer="body"`, but I don't see that string being turned into a DOM node.